### PR TITLE
Update database_driver.py

### PR DESCRIPTION
**Pull request by shrey-agarwal**
Added feature 2  (add_routeinfo())

### CODE REVIEW

File: wsgi/database_driver.py [hunk 1]
```diff
@@ -28,6 +28,11 @@ def get_userinfo(phonenumber):
         info.pop('_id', None)
     return info
 
+def add_route(phonenumber,route):
+	db = client.backend
```
Comment (sahutd) [inline wsgi/database_driver.py]: Could you use a different collection ? like student_route_info

Comment (shrey-agarwal) [inline wsgi/database_driver.py]: The dB is backend
The collection is routeinfo
You want me to change the name?

-- Regards,
Shrey Agarwal

In wsgi/database_driver.py
<https://github.com/sahutd/seteam5_backend/pull/4#discussion_r55937837>:

> @@ -28,6 +28,11 @@ def get_userinfo(phonenumber):
>          info.pop('_id', None)
>      return info
>
> +def add_route(phonenumber,route):
> +	db = client.backend

Could you use a different collection ? like student_route_info

—
Reply to this email directly or view it on GitHub
<https://github.com/sahutd/seteam5_backend/pull/4/files#r55937837>.

Comment (sahutd) [inline wsgi/database_driver.py]: @shrey-agarwal : My bad.

Comment (sahutd) [inline wsgi/database_driver.py]: @shrey-agarwal : thank you!

**Pull request merged**